source/view-model.js

```diff
@@ -4,14 +4,23 @@
 
 /*********************** View-Model **********************/
 "use strict";
+function GamePreset(){
```

Comment:
Can we put informational (non-functional) objects in the same chunk of code? GamePreset and GameInfo have the same purpose and can be placed in the code one after the other.